<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Highlight_ConfigUI</class>
 <widget class="QWidget" name="Highlight_ConfigUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>614</width>
    <height>585</height>
   </rect>
  </property>
  <layout class="QGridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="1" column="0">
    <widget class="QCheckBox" name="kcfg_HighlightSoundsEnabled">
     <property name="text">
      <string>Ena&amp;ble sound for highlights</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="KColorButton" name="kcfg_HighlightNickColor">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="KColorButton" name="kcfg_HighlightOwnLinesColor">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QCheckBox" name="kcfg_HighlightNick">
     <property name="text">
      <string>Alwa&amp;ys highlight own current nick:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QCheckBox" name="kcfg_HighlightOwnLines">
     <property name="text">
      <string>Always highlight own &amp;lines:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <spacer name="spacer11">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>120</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="2">
    <spacer name="spacer1">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>50</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="2">
    <spacer name="spacer2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>50</width>
       <height>21</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QGroupBox" name="groupBox1">
     <property name="title">
      <string>Highlight List</string>
     </property>
     <layout class="QGridLayout" name="highlightLayout">
      <item row="0" column="0">
       <widget class="HighlightTreeWidget" name="highlightListView">
        <property name="acceptDrops">
         <bool>true</bool>
        </property>
        <property name="dragEnabled">
         <bool>true</bool>
        </property>
        <property name="dragDropMode">
         <enum>QAbstractItemView::InternalMove</enum>
        </property>
        <property name="rootIsDecorated">
         <bool>false</bool>
        </property>
        <property name="allColumnsShowFocus">
         <bool>true</bool>
        </property>
        <column>
         <property name="text">
          <string>RegEx</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Highlights</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Sound</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Auto Text</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Chat Windows</string>
         </property>
        </column>
       </widget>
      </item>
      <item row="0" column="1">
       <layout class="QVBoxLayout">
        <item>
         <widget class="QPushButton" name="newButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>&amp;New</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="removeButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>&amp;Remove</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="spacer27">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>250</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QFormLayout" name="formLayout">
        <property name="fieldGrowthPolicy">
         <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="patternLabel">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>&amp;Pattern:</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
          <property name="buddy">
           <cstring>patternInput</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="KLineEdit" name="patternInput">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>1</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="colorLabel">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Color:</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="KColorButton" name="patternColor">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="soundLabel">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>&amp;Sound:</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
          <property name="buddy">
           <cstring>soundURL</cstring>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="KUrlRequester" name="soundURL">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="soundPlayBtn">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="toolTip">
             <string>Test sound</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="autoTextLabel">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>A&amp;uto text:</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
          <property name="buddy">
           <cstring>autoTextInput</cstring>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="KLineEdit" name="autoTextInput">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="chatWindowsLabel">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Chat windows:</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
          <property name="buddy">
           <cstring>chatWindowsInput</cstring>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="KLineEdit" name="chatWindowsInput">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>1</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="whatsThis">
           <string>This highlight will only be searched for in chat windows listed here by name, or in all chat windows if the field is left empty. Multiple names have to be separated by commas.</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="enableNotificationsLabel">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Enable &amp;notifications:</string>
          </property>
          <property name="buddy">
           <cstring>enableNotificationsCheckbox</cstring>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QCheckBox" name="enableNotificationsCheckbox">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
  <customwidget>
   <class>KColorButton</class>
   <extends>QPushButton</extends>
   <header>kcolorbutton.h</header>
  </customwidget>
  <customwidget>
   <class>KUrlRequester</class>
   <extends>QWidget</extends>
   <header>kurlrequester.h</header>
  </customwidget>
  <customwidget>
   <class>HighlightTreeWidget</class>
   <extends>QTreeWidget</extends>
   <header>highlighttreewidget.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>highlightListView</tabstop>
  <tabstop>newButton</tabstop>
  <tabstop>removeButton</tabstop>
  <tabstop>patternInput</tabstop>
  <tabstop>patternColor</tabstop>
  <tabstop>soundURL</tabstop>
  <tabstop>soundPlayBtn</tabstop>
  <tabstop>autoTextInput</tabstop>
  <tabstop>chatWindowsInput</tabstop>
  <tabstop>enableNotificationsCheckbox</tabstop>
  <tabstop>kcfg_HighlightSoundsEnabled</tabstop>
  <tabstop>kcfg_HighlightNick</tabstop>
  <tabstop>kcfg_HighlightNickColor</tabstop>
  <tabstop>kcfg_HighlightOwnLines</tabstop>
  <tabstop>kcfg_HighlightOwnLinesColor</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>kcfg_HighlightNick</sender>
   <signal>toggled(bool)</signal>
   <receiver>kcfg_HighlightNickColor</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>kcfg_HighlightOwnLines</sender>
   <signal>toggled(bool)</signal>
   <receiver>kcfg_HighlightOwnLinesColor</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
